lms/static/js/courseware/course_home_events.js

```diff
@@ -9,7 +9,13 @@
                 });
             });
             $('.date-summary-verified-upgrade-deadline .date-summary-link').on('click', function() {
-                Logger.log('edx.course.home.upgrade_verified.clicked', {});
+                Logger.log('edx.course.home.upgrade_verified.clicked', {location: 'sidebar'});
```

Comment:
@stroilova is there any issue with adding event data to an already existing event? There are going to be 2 places on the page (if you are routed into the experiment) where you can click to "Upgrade", and we would like to be able to differentiate between them. I could leave the pre-existing event alone and only add something to the event data for the new button, but if there's no harm, I thought it was clearer if I introduce the additional event data field for both variants.

I also struggled with the name of the new event data. "source" seemed natural, but could be confused with event_source. Likewise I considered "type", but there is already "event_type".